src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -125,7 +125,7 @@ public static class TextCatalog
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        /// [!code-csharp[FeaturizeText](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/WordEmbeddingTransform.cs)]
```

Comment:
>WordEmbeddingTransform [](start = 101, length = 22)

Should we delete this file which already has a sample for `ApplyWordEmbedding` ?

Reply:
I think it should be deleted now to avoid mess.

---
In reply to: [270972391](https://github.com/dotnet/machinelearning/pull/3142#discussion_r270972391) [](ancestors = 270972391)